src/Test.CoyoteTests/rewrite.coyote.Windows.Debug.json

```diff
@@ -0,0 +1,4 @@
+{
+  "AssembliesPath": "D:\\repos\\gcsv\\dev\\binskim\\src\\Test.CoyoteTests\\bin\\Debug\\netcoreapp3.1",
```

Comment:
>D:\\repos\\gcsv\\dev\\binskim\ [](http://example.com/codeflow?start=21&length=30)

do this file need to checkin, or add to the git ignore
will it auto create on our machines once build or